multiplatform-crypto-libsodium-bindings/src/nativeMain/kotlin/com/ionspin/kotlin/crypto/util/LibsodiumUtil.kt

```diff
@@ -30,7 +30,7 @@ actual object LibsodiumUtil {
         val result = sodium_memcmp(firstPinned.toPtr(), secondPinned.toPtr(), first.size.convert())
         firstPinned.unpin()
         secondPinned.unpin()
-        return result == 0
+        return result.isLibSodiumSuccessCode()
```

Comment:
Caught by failing build on CI, missing 'import com.ionspin.kotlin.crypto.util.isLibsodiumSuccessCode'

Reply:
Oh and also seems to be typo in capitalization `LibSodium` and it should be `Libsodium`

Reply:
Ah, thanks for catching those typos. I don't have the native targets building properly to test locally yet.